app/models/post.rb

```diff
@@ -0,0 +1,7 @@
+class Post < ActiveRecord::Base
+  has_many :comments, dependent: :destroy
+  belongs_to :user
+  has_attached_file :image, :styles => { :medium => "450x450>", :thumb => "100x100>" },
+                    :default_url => "missing.png"
+  validates_attachment_content_type :image, :content_type => /\Aimage\/.*\Z/
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)<br>Style/RegexpLiteral: Use `%r` around regular expression. (https://github.com/bbatsov/ruby-style-guide#percent-r)